[2024-06-26T22:29:45.340+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-26T22:29:45.360+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transfer_to_mysql.load_data_to_mysql manual__2024-06-26T22:29:44.455785+00:00 [queued]>
[2024-06-26T22:29:45.366+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transfer_to_mysql.load_data_to_mysql manual__2024-06-26T22:29:44.455785+00:00 [queued]>
[2024-06-26T22:29:45.367+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-26T22:29:45.375+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data_to_mysql> on 2024-06-26 22:29:44.455785+00:00
[2024-06-26T22:29:45.385+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_transfer_to_mysql', 'load_data_to_mysql', 'manual__2024-06-26T22:29:44.455785+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/data_transfer_dag.py', '--cfg-path', '/tmp/tmpjdcv74oy']
[2024-06-26T22:29:45.387+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask load_data_to_mysql
[2024-06-26T22:29:45.388+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1032) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-26T22:29:45.388+0000] {standard_task_runner.py:63} INFO - Started process 1034 to run task
[2024-06-26T22:29:45.422+0000] {task_command.py:426} INFO - Running <TaskInstance: data_transfer_to_mysql.load_data_to_mysql manual__2024-06-26T22:29:44.455785+00:00 [running]> on host fa313e9b9eb9
[2024-06-26T22:29:45.501+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_transfer_to_mysql' AIRFLOW_CTX_TASK_ID='load_data_to_mysql' AIRFLOW_CTX_EXECUTION_DATE='2024-06-26T22:29:44.455785+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-26T22:29:44.455785+00:00'
[2024-06-26T22:29:45.501+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-26T22:29:45.520+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2024-06-26T22:29:46.487+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-26T22:29:46.488+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-26T22:29:46.494+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_transfer_to_mysql, task_id=load_data_to_mysql, run_id=manual__2024-06-26T22:29:44.455785+00:00, execution_date=20240626T222944, start_date=20240626T222945, end_date=20240626T222946
[2024-06-26T22:29:46.524+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-26T22:29:46.545+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-26T22:29:46.547+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
